### PR TITLE
Add more information on the homepage

### PR DESCRIPTION
**Pull request by MoOx**
Preview
## screen shot 2016-08-04 at 23 08 11

screen shot 2016-08-04 at 23 04 28

### CODE REVIEW

**Comment by hzoo** [comment]
Awesome

File: index.html [hunk 1]
```diff
@@ -92,6 +92,12 @@
         <h2>
           <a href="#commands">Commands</a>
         </h2>
+        
+        <p>
+          Here is a brief introduction for each commands.
+          Please visit the <a href="https://github.com/lerna/lerna#readme">README</a>
+          for more detailed informations.
```
Comment (hzoo) [inline index.html:99]: Do we want to mention this isn't up to date or is this good enough to say?

Comment (MoOx) [inline index.html:99]: I guess this notice is a warning :)

File: index.html [hunk 2]
```diff
@@ -119,6 +125,11 @@
           <p>
             Bootstrap the packages in the current Lerna repo. Installing all their
             dependencies and linking any cross-dependencies.
+            <strong>
+              This command is crucial as it allows you to use your package
+              names in <code>require()</code> as if the packages where already existing
+              and available in your <code>node_modules</code> folder.
+            </strong>
```
Comment (hzoo) [inline index.html:134]: s/where/were

maybe make it under another `<p>`?

Comment (MoOx) [inline index.html:134]: Oh I forgot the `<br />` :D

Comment (MoOx) [inline index.html:134]: Fixed.

**Comment by hzoo** [comment]
cool 👍

**Pull request merged**